### ISSUE TITLE
Login to newly created wireguard app not working

### ISSUE THREAD

**Issue by sebthom** [hetznercloud/apps#114]
### TL;DR

After provisioning a new server with the wireguard app, a login via browser is not possible. Each login attempt simply redirects to the login page itself.

I found this issue https://github.com/ngoduykhanh/wireguard-ui/issues/641 without further details. The wireguard-ui project seems dead. Last update 9 months ago. Also from a security point of view it might make sense that you replace wireguard-ui with e.g. https://github.com/wg-easy/wg-easy which is actively maintained.

### Expected behavior

The login works.

### Observed behavior

1. Open the wg admin ui in the browser
2. try to login using the preselected user/pw
3. Click "Sign-In"
4. The page reloads and shows the login page again.
5. In the script console you see:
   ```js
	Uncaught TypeError: data is undefined
	    success https://wireguard.myserver.com/login?next=/:118
	    jQuery 6
	    <anonymous> https://wireguard.myserver.com/login?next=/:110
	    jQuery 7
	    <anonymous> https://wireguard.myserver.com/login?next=/:98
	    jQuery 8
	    <anonymous> https://wireguard.myserver.com/login?next=/:96
	    jQuery 13
	login:118:94
   ```

**Comment by Kjarrigan** [comment]
Thanks for your report @sebthom - other users have mentioned that issue too and it seems like there is a container restart necessary after the admin user was create before the UI login properly works. We'll check an automatic solution in the setup.

I'll also forward your suggestion regarding wg-easy to our internal apps maintainer.
